libs/ui/src/Admin/PageHeader.tsx

```diff
@@ -0,0 +1,166 @@
+import { FC, useState } from 'react'
+
+import { SearchIcon } from '@chakra-ui/icons'
+import {
+  Flex,
+  Avatar,
+  Input,
+  Text,
+  Button,
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+} from '@chakra-ui/react'
+import { HiOutlineFilter } from 'react-icons/hi'
+import { VscListFilter } from 'react-icons/vsc'
+
+interface PageHeaderProps {
+  onSearch: (item: string) => void
+  title: string
+  images: { slug: string; url: string }[]
+  textColor: string
+  backgroundColor: string
+  createNew: () => void
+}
+
+export const PageHeader: FC<PageHeaderProps> = ({
+  title,
+  onSearch,
+  images,
+  textColor,
+  backgroundColor,
+  createNew,
+}) => {
+  const [search, setSearch] = useState('')
+  const [focusColor, setFocusColor] = useState('lightgray')
+  const [flags, setFlags] = useState('tr')
+
+  return (
+    <Flex
+      justifyContent={{ base: 'space-between' }}
+      alignItems={'center'}
+      padding={1}
+      h={{ base: '60px' }}
+      backgroundColor={backgroundColor}
+      borderRadius={{ base: '10px' }}
+    >
+      <Text color={textColor} w={{ base: '40%' }} fontWeight={'bold'}>
```

Comment:
Width is needed here?: 

Also, `base` applies from 0px bearkpoint if you don't give any other breakpoints. It's probably the same think as `w="40%"`. You can read an explanation about responsive styles here: https://github.com/wsvvrijheid/monorepo/pull/47#discussion_r944140256

```suggestion
      <Text color="primary.400"  fontWeight={'bold'}>
```